src/reset.css

```diff
@@ -0,0 +1,133 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 

```

Comment:
in general it is a good practice to add reset.css, though in your task it is a bit redundant.